x-pack/test/osquery_cypress/runner.ts

```diff
@@ -75,6 +75,22 @@ export async function OsqueryCypressVisualTestRunner(context: FtrProviderContext
   await startOsqueryCypress(context, 'open');
 }
 
+export async function OsqueryCypressServerlessTestRunner(context: FtrProviderContext) {
+  const esClient = context.getService('es');
+
+  await esClient.indices.putSettings({
+    index: '*,.*,-.security*,-.fleet-*,-.async-search*,-.transform-internal-*,-.tasks*',
```

Comment:
why are the other patterns needed, doesn't the `*` cover all indices?

Reply:
it's following the guideline provided by the Productivity Engineering Team